fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/AppAndSystemHelper.kt

```diff
@@ -60,46 +61,85 @@ object AppAndSystemHelper {
         }
     }
 
-    @RequiresApi(Build.VERSION_CODES.R)
+    /**
+     * Checks if a specific download file is inside the device storage and deletes it.
+     *
+     */
```

Comment:
I would add a comment here detailing the API differences, and why we need to check.

Reply:
Done.